static/js/markdown.js

```diff
@@ -136,9 +136,8 @@ exports.add_topic_links = function (message) {
 
 exports.is_status_message = function (raw_content, content) {
     return raw_content.indexOf('/me ') === 0 &&
-            raw_content.indexOf('\n') === -1 &&
             content.indexOf('<p>') === 0 &&
-            content.lastIndexOf('</p>') === content.length - 4;
+            content.indexOf('</p>') !== -1;
```

Comment:
Why is this the correct new test?

Reply:
As there can be multiple "block" objects in a `/me` message, index of `</p>` cannot be specified, as it may vary from message to message, say for example:
```
/me first line
> some quote here
```
will be rendered as
```html
<p>/me first line</p><blockquote>some quote here</blockquote>
```
Here it cannot be specifically determined the position of `</p>` We cannot even use a hack of the sort like checking the index of `\n` in the markdown message as the following can cause a problem...
```
/me first line
second line
```
gets converted to
```html
<p>/me first line<br>second line</p>
```
So, determining the position of `</p>` was removed from the test and it was just confirmed to be present. Though, thinking about it, this might be redundant. Should I remove this, I guess there's no harm in keeping or removing this test.